src/Mode.h

```diff
@@ -471,10 +507,15 @@ inline byte_vec_t pack_native_color(const rgba_t color, Mode mode) {
     v.push_back(color ^ 0x07);
     break;
   case Mode::wsc:
+  case Mode::gg:
   case Mode::wsc_packed:
     v.push_back(((color >> 16) & 0x0F) | ((color >> 4) & 0xF0));
     v.push_back((color & 0x0F));
     break;
+  case Mode::sms:
+    v.push_back(((color >> 12) & 0x30) | ((color >> 6) & 0x0C) | (color & 3));
+    break;
+  case Mode::pv1k:
```

Comment:
What's this mode? 🤔

Reply:
It's how you know I didn't re-build after cherry-picking the SMS support away from the PV-1000 support :sigh: